ext_emconf.php

```diff
@@ -32,8 +32,8 @@
 	'version' => '4.3.3',
 	'constraints' => array(
 		'depends' => array(
-			'php' => '5.4.0-5.6.99',
```

Comment:
Let's add testing etc. for that and require passing if it works - I'll make another PR!